+++
title = "Endophytic microbiome variation at the level of a single plant seed"
date = 2021-04-27T00:00:00

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Ari Fina Bintarti", "Abby Sulesky-Grieb", "__Nejc Stopnisek__", "Ashley Shade"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Preprint / Working Paper
# 4 = Report
# 5 = Book
# 6 = Book section
# 7 = Thesis
# 8 = Patent
publication_types = ["3"]

# Publication name and optional abbreviated version.
publication = "bioRxiv"
publication_short = ""

# Abstract.
abstract = ""

# Summary. An optional shortened abstract.
summary = "Like other plant compartments, the seed harbors a microbiome. The members of the seed microbiome are the first to colonize a germinating seedling, and they initiate the trajectory of microbiome assembly for the next plant generation. Therefore, the members of the seed microbiome are important for the dynamics of plant microbiome assembly and the vertical transmission of potentially beneficial symbionts. However, it remains challenging to assess the microbiome at the individual seed level (and, therefore, for the future individual plant) due to low endophytic microbial biomass, seed exudates that can select for particular members, and high plant and plastid contamination of resulting reads. Here, we report a protocol for extracting metagenomic DNA from an individual seed (common bean, Phaseolus vulgaris L.) with minimal disruption of host tissue, which we expect to be generalizable to other medium-and large-seed plant species. We applied this protocol to quantify the 16S rRNA V4 and ITS2 amplicon composition and variability for individual seeds harvested from replicate common bean plants grown under standard, controlled conditions to maintain health. Using metagenomic DNA extractions from individual seeds, we compared seed-to-seed, pod-to-pod, and plant-to-plant microbiomes, and found highest microbiome variability at the plant level. This suggests that several seeds from the same plant could be pooled for microbiome assessment, given experimental designs that apply treatments at the maternal plant level. This study adds protocols and insights to the growing toolkit of approaches to understand the plant-microbiome engagements that support the health of agricultural and environmental ecosystems."

# Digital Object Identifier (DOI)
doi = "10.1101/2021.04.27.441449"

# Is this a featured publication? (true/false)
featured = false

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["plant-microbe"]

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["plant-microbe"]

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides = ""

# Links (optional).
url_pdf = "https://www.biorxiv.org/content/10.1101/2021.04.27.441449v1.full.pdf"
url_code = "https://github.com/ShadeLab/Bean_seed_variability_Bintarti_2021"
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# links = [{name = "Custom Link", url = "http://example.org"}]

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = "Smart"
+++
